### DISCUSSION TITLE
Reverse proxy https vs nextcloud e

### DISCUSSION THREAD

**Discussion by sunmetis** [nextcloud/all-in-one#5674] [Questions] [answered]
Hi . I m using traefik. Finally I can connect to nextcloud wiith all declared middlewares and with https to the interface BUT i cannot put the domain during installation (probably https alert). With docker i m using ports 8443 and 8080. I can only connect to the 8080. Nothing to do. The funny thing : even when using 8080 im https /certificate how to use it ?? Please

**Comment by szaimen** [comment, marked as answer]
Hi, does this video help you? https://www.youtube.com/watch?v=VLPSRrLMDmA

**Comment by sunmetis** [comment]
Hi thx for answer . Finally using nextcloud "standard " not AIO . Its
working exept for middlewares X-ROBOTS. Do you have discord

Discord : SUNMETIS
Anthony COPPET
0696 70 13 81
